tests/puppeteer/pages/Install/install.js

```diff
@@ -133,6 +143,16 @@ module.exports = class Install extends CommonPage {
    */
   async checkInstallationSuccessful() {
     await this.page.waitForSelector(this.installationProgressBar, {visible: true});
+    await this.page.waitFor(this.installationProgressBar, {visible: true});
+    await this.page.waitFor(this.generateSettingsFileStep, {visible: true});
+    await this.page.waitFor(this.installDatabaseStep, {visible: true});
+    await this.page.waitFor(this.installDefaultDataStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.populateDatabaseStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.configureShopStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installModulesStep, {visible: true});
+    await this.page.waitFor(this.installModulesAddons, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installThemeStep, {visible: true});
+    await this.page.waitFor(this.installFixturesStep, {visible: true});
     await this.page.waitForSelector(this.finalStepPageTitle, {visible: true, timeout: 90000});
     await this.checkStepTitle(this.finalStepPageTitle, this.finalStepEnTitle);
```

Comment:
Since you're using the method `checkStepTitle` only once, maybe it's not really necessary to have it as a separate method ?

Reply:
@SimonGrn the method `checkStepTitle` is used many times in `installShop.js`

Reply:
Using a function only once is not necessarily a problem, sometimes it's nice to encapsulate a few instructions in a function, and give it a descriptive name, so that the reader immediately knows what the coder intended to do when he sees the function call.

But that's just my opinion, not really an official good practice :D

Reply:
Damn I got roasted on this one :laughing: 
That's good for me then !

Reply:
haha, I didn't mean it that way :D